static/app/views/replays/deadRageClick/exampleReplaysList.tsx

```diff
@@ -0,0 +1,113 @@
+import {Fragment, useMemo} from 'react';
+import {Location} from 'history';
+
+import EmptyStateWarning from 'sentry/components/emptyStateWarning';
+import LoadingIndicator from 'sentry/components/loadingIndicator';
+import {t} from 'sentry/locale';
+import EventView from 'sentry/utils/discover/eventView';
+import getRouteStringFromRoutes from 'sentry/utils/getRouteStringFromRoutes';
+import useReplayList from 'sentry/utils/replays/hooks/useReplayList';
+import useOrganization from 'sentry/utils/useOrganization';
+import {useRoutes} from 'sentry/utils/useRoutes';
+import {StatusContainer} from 'sentry/views/profiling/landing/styles';
+import {ReplayCell} from 'sentry/views/replays/replayTable/tableCell';
+
+function transformSelectorQuery(selector: string) {
+  return selector
+    .replaceAll('"', `\\"`)
+    .replaceAll('aria=', 'aria-label=')
+    .replaceAll('testid=', 'data-test-id=');
+}
+
+export default function ExampleReplaysList({
+  selector,
+  location,
+  clickType,
+  deadOrRage,
+}: {
+  clickType: 'count_dead_clicks' | 'count_rage_clicks';
+  deadOrRage: 'dead' | 'rage';
+  location: Location;
+  selector: string;
+}) {
+  const organization = useOrganization();
+  const query = transformSelectorQuery(selector);
+  const {project, environment, start, statsPeriod, utc, end} = location.query;
+  const emptyLocation: Location = useMemo(() => {
+    return {
+      pathname: '',
+      search: '',
+      hash: '',
+      state: '',
+      action: 'PUSH' as const,
+      key: '',
+      query: {project, environment, start, statsPeriod, utc, end},
+    };
+  }, [project, environment, start, statsPeriod, utc, end]);
+
+  const eventView = useMemo(
+    () =>
+      EventView.fromNewQueryWithLocation(
+        {
+          id: '',
+          name: '',
+          version: 2,
+          fields: [
+            'activity',
+            'duration',
+            'id',
+            'project_id',
+            'user',
+            'finished_at',
+            'is_archived',
+            'started_at',
+            'urls',
+          ],
+          projects: [],
+          query: `${deadOrRage}.selector:"${query}"`,
```

Comment:
what if we passed in the whole query, hoist all this up including `transformSelectorQuery`